app/models/base.py

```diff
@@ -0,0 +1,10 @@
+from app.models import db
+
+
+class SoftDeletionModel(db.Model):
```

Comment:
The motive behind inheriting from this model is to avoid adding `deleted_at` attributes for all the models that support soft deletion, right? Are there other advantages of having a whole new model just for soft deletion?

Reply:
Yep, just so that we don't repeat code. There won't be a database table associated with this though.

Reply:
maybe you can as well abstract it even further say, using a common open-event schema, but maybe in the future.